### PR TITLE
Feature/main section2

### PR DESCRIPTION
**Pull request by sasigit7**
Dear Toucans, Done basic layout with HTML and CSS for MainSection2. Please have a look and let me know your feedback.

### CODE REVIEW

**Review by stephairth — COMMENTED** [review]
Just deleted an extra </head> in the HTML. When I open index.html the images don't load for me, but the addresses check out so not totally sure what's going on there. But overall looks good! 🙆‍♀️

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+.bigImages {
+    display: flex;
+    align-items: center;
+    flex-flow: row nowrap;
+    flex-direction: column;
+}
+
+.cardImgOne>.img {
+    height: 450px;
+    background: url("/images/bigImgOne.jpg");
```
Comment (ahmedserifoski) [inline css/style.css]: I can't see the imags
try doing this instead  background: url("../images/bigImgOne.jpg");

**Review by ahmedserifoski — COMMENTED** [review]
I can't see the images, I left u a comment down below
the scrolling bar isn't supposed to be there while hovering over an image
also, try to merge it into development, not master

**Review by VladimirtheGreatest — CHANGES_REQUESTED** [review]
Yea, make sure you have a look at those images I cant see them as well and do not merge into master branch yet.  We already have a navigation bar, why did you not include it?

**Comment by sasigit7** [comment]
> Just deleted an extra in the HTML. When I open index.html the images don't load for me, but the addresses check out so not totally sure what's going on there. But overall looks good! 🙆‍♀️

Hmm! That's weird. Images are loading and I can see them with the same files on my local machine though.  Anyway, I will have a look and find out where it's going wrong. Thanks @stephairth.

**Comment by sasigit7** [comment]
> Yea, make sure you have a look at those images I cant see them as well and do not merge into master branch yet. We already have a navigation bar, why did you not include it?

Sure! will do.  I merged it by mistake and dint realised until later. Also, I have to do some refactoring and write some media queries too.  

Why I didn't include navigation bar code to my code? Well, we discussed this in today's meeting which you have missed and we have a reason. lol.  Plz check in discord for full explanation.

**Comment by sasigit7** [comment]
> I can't see the images, I left u a comment down below
> the scrolling bar isn't supposed to be there while hovering over an image
> also, try to merge it into development, not master

->haha. You have the same Image issue. You and steph saw the demo in Zoom and it's all working right. Never mind, that's the common issue for everyone - So, I will definitely fix it. 
->Scrolling bar will be gone soon - I promise. 
-> Yes sir, While I was merging files-  we three didn't noticed it but we realised later right. That will be done.

**Review by stephairth — COMMENTED** [review]
Looks good to me! 🙆‍♀️

**Review by stephairth — APPROVED** [review]

**Comment by VladimirtheGreatest** [comment]
That slick carousel looks great.

**Review by ahmedserifoski — COMMENTED** [review]
The images r still getting stretched and squeezed

**Review by VladimirtheGreatest — COMMENTED** [review]
Looks great just finish the media queries for mobiles and we are good to go, I can actually help you with that.

**Review by stephairth — CHANGES_REQUESTED** [review]
Looking good! A few nitpicks: 
1) The images look a bit vertically stretched:

a bit stretched

Compared to the original: 
less stretchy original site

2) The original has a dark gradient along the bottom of the larger top images that makes the white font easier to read. I think it would be good to add this if you weren't already planning to 👍

3) The zoom on the smaller images is making their height increase, which doesn't happen on the original. Since this doesn't happen on the larger images above, I think it shouldn't happen to the smaller ones for consistency.

File: css/style.css (hunk 1 above)
Comment (sasigit7) [reply css/style.css]: This has been fixed a while ago and two dots before / worked. Thanks @ahmedserifoski :+1:

**Comment by sasigit7** [comment]
> Just deleted an extra in the HTML. When I open index.html the images don't load for me, but the addresses check out so not totally sure what's going on there. But overall looks good! 🙆‍♀️

Thanks, @stephairth.  The issue has been resolved. :+1:

**Comment by sasigit7** [comment]
> Yea, make sure you have a look at those images I cant see them as well and do not merge into master branch yet. We already have a navigation bar, why did you not include it?

It's been fixed a while ago.

**Comment by sasigit7** [comment]
> Looking good! A few nitpicks:
> 
> 1. The images look a bit vertically stretched:
> 
> a bit stretched
> 
> Compared to the original:
> less stretchy original site
> 
> 1. The original has a dark gradient along the bottom of the larger top images that makes the white font easier to read. I think it would be good to add this if you weren't already planning to 👍
> 2. The zoom on the smaller images is making their height increase, which doesn't happen on the original. Since this doesn't happen on the larger images above, I think it shouldn't happen to the smaller ones for consistency.

All issues have been fixed.

**Review by stephairth — APPROVED** [review]

**Review by VladimirtheGreatest — APPROVED** [review]
Good job.

**Review by ahmedserifoski — APPROVED** [review]

**Pull request merged**